pkgs/shells/powershell/getHashes.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p bash wget coreutils gnutar nix
+version=$1
+
+if [[ -z "$1" ]]
+then
+    echo "Pass the version to get hashes for as an argument"
+    exit 1
+fi
+
+allOutput=""
+
+for plat in osx linux
+do
+    for arch in x64 arm64
+    do
+        dlDest=$(mktemp)
+        exDest=$(mktemp -d)
```

Comment:
We should only create those once and add a TRAP to cleanup up after ourselves.

Reply:
I don’t know how to do that.

Reply:
https://github.com/NixOS/nixpkgs/blob/634141959076a8ab69ca2cca0f266852256d79ee/pkgs/applications/networking/instant-messengers/element/seshat/update.sh#L21

Reply:
OK, I did that